tools/SourceKit/CMakeLists.txt

```diff
@@ -93,6 +93,12 @@ include_directories(BEFORE
 if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
   set(SOURCEKIT_DEFAULT_TARGET_SDK "LINUX")
   if(SWIFT_BUILD_SOURCEKIT)
+    if(SWIFT_BUILD_STDLIB)
+      set(libdispatch_ENABLE_SWIFT YES)
```

Comment:
Should `libdispatch_ENABLE_SWIFT` include the `SOURCEKIT` prefix ? It seems it has potential to get confused with some other global variable.

Reply:
Sure!  I'll adjust that.